packages/financial-templates-lib/src/logger/PagerDutyV2Transport.ts

```diff
@@ -3,18 +3,22 @@ import Transport from "winston-transport";
 import { event } from "@pagerduty/pdjs";
 import * as ss from "superstruct";
 
+import { TransportError } from "./TransportError";
+
 type TransportOptions = ConstructorParameters<typeof Transport>[0];
 export type Severity = "critical" | "error" | "warning" | "info";
 export type Action = "trigger" | "acknowledge" | "resolve";
 
 const Config = ss.object({
   integrationKey: ss.string(),
   customServices: ss.optional(ss.record(ss.string(), ss.string())),
+  logTransportErrors: ss.optional(ss.boolean()),
 });
 // Config object becomes a type
 // {
 //   integrationKey: string;
 //   customServices?: Record<string,string>;
+//   logTransportErrors?: boolean;
```

Comment:
Do we need this comments?

Reply:
Not strictly required, but they provide explicit type descriptions that are not otherwise readable in the code